nodejs-multiselect/nodejs-multiselect.spec

```diff
@@ -0,0 +1,58 @@
+%global npm_name multiselect
+%global enable_tests 1
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 0.9.12
+Release: 1%{?dist}
+Summary: A user-friendlier drop-in replacement for the standard select with multiple attribute activated
+License: WTFPL
```

Comment:
The licence text shipped inside the package is MIT, I don't think the metadata is correct.